dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/csi/EJBApplicationMetaData.java

```diff
@@ -198,25 +202,38 @@ public class EJBApplicationMetaData {
     // F743-506
     private List<TimerNpImpl> ivQueuedNonPersistentTimers;
 
+    /**
+     * List of Runnable instances that will preload bean pools for those stateless beans that
+     * have been configured with a hard minimum pool size. Additionally, stateless bean pools
+     * may be preloaded when the server has been started for taking a checkpoint.
+     *
+     * Bean pools will be preloaded at the end of application start after all startup bean
+     * post construct methods have been called and the application will be unblocked for work
+     * to to ensure all beans in the application have been identified (in case there are bean
+     * references) and to ensure the application state has been properly initialized before
+     * running any additional application code that may occur during the preload.
```

Comment:
@brideck Agree this is hard to follow.  I've removed the duplicate copy of the comment on the instance variable declaration, and updated the version on the add method similar to your suggestions, as follows:

```
     * To ensure all beans in the application have been identified (in case there are bean
     * references that need to be resolved prior to instance creation) and the application's
     * state has been properly initialized (before running any additional application code that
     * may occur during the preload), bean pools will be preloaded at the end of application
     * start, after all beans have been identified, all startup bean post construct methods
     * have been called, and the application has been unblocked for work.
```